README.md

```diff
@@ -67,6 +66,7 @@ imagelab.report(issue_types=issue_types)
 - [Additional example notebooks](https://github.com/cleanlab/cleanvision-examples)
 - [Documentation](https://cleanvision.readthedocs.io/)
 - [Blog Post](https://cleanlab.ai/blog/cleanvision/)
+- [FAQ](FAQ.md)
 
```

Comment:
the FAQ needs to be linked from the documentation side bar too right?

Reply:
why have the FAQ be in the GH repo instead of in the documentation website?  Normally FAQ would be in the documentation website

Reply:
Including in the docs